content/docs/roadmap/v1.2.0.md

```diff
@@ -0,0 +1,53 @@
++++
```

Comment:
Full list of all features:

## Core
- [Enable NSE to inject Policy Based Routes into NSC](https://github.com/networkservicemesh/api/issues/119)
- [Enable Mutually Aware NSEs to overlap in IPAM/Routes](https://github.com/networkservicemesh/api/issues/121) 
## NSM Features
- vl3
- nsmctl to allow for non-K8s manipulation of registry
- Initial implementation p2mp implementation
- Calico-VPP Integration
- Open telemetry integration
- Exclude prefixes already in use in a Pod by other Network Services
- Advanced healing (death/restart of 2 components)
- Advanced registry healing (death/restart remote registry chain)
- Advanced healing - condition healing on datapath being down cmd-forwarder-vpp
- AF_XDP support
## Examples
- Forwader selection example
- Add more floating interdomain/interdomain examples
- IP payload
- Scale from zero vl3 example
- vl3 examples
- Add NSM examples when spire is unhealthy
- Run OVS Examples in CI
- Fuzz testing
## Stability
- Improve releasing mechanics
- Improve system stability (reduce bugs count)
## Usability
- Architects
- Admins
- Users
- Enterprise
- SP
- NSE Developers
- SigStore image signing
## Performance
- Calico-VPP+NSM dataplane performance testing on Equinix Metal